app/src/main/java/com/oracle/javafx/scenebuilder/app/preferences/PreferencesRecordGlobal.java

```diff
@@ -456,6 +466,9 @@ public void readFromJavaPreferences() {
         // Accordion animation
         setAccordionAnimation(applicationRootPreferences.getBoolean(ACCORDION_ANIMATION, true));
 
+        // Wildcard imports
+        setWildcardImports(applicationRootPreferences.getBoolean(WILDCARD_IMPORT, false));
```

Comment:
Use the field `DEFAULT_WILDCARD_IMPORTS` instead of hardcoding to `false`? The same can be done for `ACCORDION_ANIMATION` above.